tests/test_verifyStats.py

```diff
@@ -186,11 +184,11 @@ def setUp(self):
 
         # Use this to test the metadataStats code, as this is the case
         # it's designed to fix.
-        metadataContents = PropertySet()
-        metadataContents.setFloat("RESIDUAL STDEV C:0,0", 12.0)
-        metadataContents.setFloat("RESIDUAL STDEV", 24.0)
-        self.metadata = PropertySet()
-        self.metadata.setPropertySet("subGroup", metadataContents)
+        metadataContents = {}
```

Comment:
I guess this is the least-functional type of metadata that could be set, and hence the lowest common denominator, but when `TaskMetadata` is available, wouldn't it be better to use it instead to potentially provide a more complete and accurate test?

Reply:
Yes, but I didn't want to have the test code using the internal private state of pipe_base.

Reply:
Although of course I can just use `TaskMetadata` because when I merge pipe_base it will be available even if it's not used by `Task` by default.